packages/vue/src/vue-component-lib/utils.ts

```diff
@@ -1,7 +1,7 @@
 import { VNode, defineComponent, getCurrentInstance, h, inject, ref, Ref } from 'vue';
 
-export interface InputProps {
```

Comment:
You will need to install dependencies for `core` to be up-to-date with the latest patch from output targets. Then re-run `npm run build`. This change shouldn't add `extends Object` back in.